src/dotnet/Chat.UI.Blazor/Components/LeftPanel/LeftSearchPanel/RecentlyViewedGroups.razor

```diff
@@ -0,0 +1,44 @@
+@namespace ActualChat.Chat.UI.Blazor.Components
+@inherits ComputedStateComponent<IReadOnlyList<ChatInfo>>
+
+@{
+    var chats = State.ValueOrDefault ?? [];
+    if (chats.Count == 0)
+        return;
+}
+
+<div class="recently-viewed-groups">
+    <div class="c-header">
+        <div class="c-title">Recently viewed</div>
+        @* <HeaderButton Class="blackout"> *@
```

Comment:
Do you still need this markup in comments?

Reply:
removed